### PR TITLE
Update phpspec

### PR DESCRIPTION
**Pull request by peter-gribanov**
Update phpspec for use this library on PHP 7.x

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
         "doctrine/orm": ">=2.2.3"
     },
     "require-dev": {
-        "phpspec/phpspec": "~2.1"
+        "phpspec/phpspec": "^2.5"
```
Comment (Nyholm) [inline composer.json]: We should also allow 2.1 or older PHP versions will not work on travis. 

`^2.1`

Comment (peter-gribanov) [inline composer.json]: Already noticed

**Comment by peter-gribanov** [comment]
I apologize for the large number of commits.
There is no way to verify the support of all platforms locally.

**Comment by Nyholm** [comment]
No worries. That is why we have Travis.

Thank you for your contritions.

File: .travis.yml [hunk 2]
```diff
@@ -5,13 +5,13 @@ php:
...
   - 5.6
   - 7.0
+  - 7.1
   - hhvm
 
 matrix:
   fast_finish: true
   allow_failures:
     - php: hhvm
-    - php: 7.0
```
Comment (Nyholm) [inline .travis.yml]: Good!

**Pull request merged**